docs/pages/build/building-from-github.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Trigger builds from the Expo GitHub App
+description: Learn how to trigger builds on EAS for your app using the Expo GitHub App.
+---
+
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+
+This document outlines how to trigger builds on EAS for your app from a CI environment such as GitHub Actions, Travis CI, and more.
+
+Before building with EAS on CI, we need to install and configure `eas-cli`. Then, we can trigger new builds with the `eas build` command.
+
+## Prerequisites
+
+> While in preview, this feature is only available to EAS subscribers.
+
+### Run a successful build from your local machine
+
+To trigger EAS builds from a GitHub, we first need to configure our app for EAS Build and successfully run a build from our local machine for each platform that we'd like to support on GitHub.
+
+If you have run `eas build -p [all|ios|android]` successfully before, then you can continue.
+
+If you haven't done this yet, please refer to the [Creating your first build](setup.mdx) guide and return here when you're ready.
```

Comment:
```suggestion
If you have run `eas build -p [all|ios|android]` successfully before, then you can continue. If you haven't done this yet, refer to the [Create your first build](setup.mdx) and return here when ready.
```

We can omit the word "please" since this is an instruction.